index.js

```diff
@@ -54,15 +55,24 @@ app.post('/addComment', (request, response) => {
 	const username = request.body.username;
 
 	let msg = db.add({ username, comment });
+	
 	if (msg) {
 		logger.error(msg);
 		msg = 'There was an error.';
 	} else {
 		msg = 'Thanks for commenting!';
 	}
-	return response.redirect(`/?message= + ${msg}`);
+	
+	const comments = db.get();
+	const commentsHTML = nunjucks.render('comments.njk', {comments});
+	return response.send({commentsHTML, msg});
 });
 
+app.get('/comments', (request, response) => {
+	const comments = db.get();
+	return response.send({ comments })
+})
+
```

Comment:
Is this dead code?

Reply:
Well, what if we need to handle this request? 😄 Yeah, I guess it's dead.